modules/home/assets/js/components/home-screen.js

```diff
@@ -19,7 +19,7 @@ const HomeScreen = ( props ) => {
 				/>
 				<Box sx={ { display: 'flex', flexDirection: { xs: 'column', sm: 'row' }, justifyContent: 'space-between', gap: 3 } }>
 					<Stack sx={ { flex: 1, gap: 3 } }>
-						<GetStarted getStartedData={ props.homeScreenData.get_started } />
+						<GetStarted getStartedData={ props.homeScreenData.get_started[ 0 ] } />
```

Comment:
I think this needs to be updated since Elena's latest update:

```suggestion
						<GetStarted getStartedData={ props.homeScreenData.get_started } />
```